src-electron/importexport/import-isc.js

```diff
@@ -368,9 +368,16 @@ async function loadEndpointType(db, sessionId, packageId, endpointType) {
 
   if (dev == null)
     throw new Error(`Unknown device type: ${deviceName} / ${deviceCode}`)
+  // Get session partition given the device type reference
+  let sessionPartitionInfo =
+    await querySession.selectSessionPartitionInfoFromDeviceType(
+      db,
+      sessionId,
+      dev.id
+    )
   return queryConfig.insertEndpointType(
     db,
-    sessionId,
+    sessionPartitionInfo[0],
```

Comment:
why hardcoded 0?

Reply:
Since selectSessionPartitionInfoFromDeviceType returns an array and import disc does not have a cap use case it is safe to pick the first one in this case.